lib/validator/validator.go

```diff
@@ -34,7 +34,7 @@ var validate *validator.Validate
 var (
 	nameRegex          = regexp.MustCompile("^[a-zA-Z0-9_.-]{1,128}$")
 	interfaceRegex     = regexp.MustCompile("^[a-zA-Z0-9_-]{1,15}$")
-	labelRegex         = regexp.MustCompile("^([a-zA-Z0-9.-]{0,253}/)?[a-zA-Z0-9]([a-zA-Z0-9_.-]{0,61}[a-zA-Z0-9])?$")
+	labelRegex         = regexp.MustCompile(`^([a-zA-Z0-9\_\.\-\/]{0,252}[a-zA-Z0-9])?$`)
```

Comment:
As should `-`, I think.

Also, looks like `.` doesn't need to be escaped either: https://regex101.com/r/PH8aQM/1